lib/licensee/license.rb

```diff
@@ -143,6 +143,19 @@ def inspect
       "#<Licensee::License key=#{key}>"
     end
 
+    # Returns an array of strings of substitutable fields in the license body
+    def fields
+      @fields ||= LicenseField.from_content(content)
+    end
+
+    # Returns a string with `[fields]` replaced by `{{{fields}}}`
+    # Does not mangle non-supported fields in the form of `[field]`
```

Comment:
I see how you've accomplished this by building a regex matching any of the fields specified in choosealicense.com's field metadata, in `license_field.rb`. :party: